TDD-IntegrationTests/HealthCheckIntegrationTest.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Microsoft.AspNetCore.Mvc.Testing;
+using System.Net;
+using TDDProject;
+
+namespace Integration
```

Comment:
How namespace is different from Project name

Reply:
changed